ContinuousRegistration/Source/util.py

```diff
@@ -12,16 +12,30 @@ def load_submissions(parameters):
     logging.info('Loading blueprints.')
 
     submissions = dict()
-    team_names = [team_name for team_name in os.listdir(parameters.submissions_directory)
-                  if os.path.isdir(os.path.join(parameters.submissions_directory, team_name))]
-    for team_name in team_names:
-        blueprints = []
-        for blueprint_name in os.listdir(os.path.join(parameters.submissions_directory, team_name)):
-            if blueprint_name.endswith('.json') or blueprint_name.endswith('.xml'):
-                logging.info('Found blueprint %s/%s.' % (team_name, os.path.basename(blueprint_name)))
-                blueprints.append(os.path.join(parameters.submissions_directory, team_name, blueprint_name))
+    if parameters.team_name is not None:
+        team_names = parameters.team_name
```

Comment:
For my understanding, should `parameters.team_name` be renamed to `parameters.team_names` (with an `s')?

Reply:
The name is set by argparse from the command line interface. There might be one arg `--team-name TeamElastix` or there might be more args `--team-name TeamElastix --team-name other-team`. If `parameters.team_name` was renamed to `parameters.team_names` users would see `--team-names TeamElastix` :/

The s here is to make the following loop readable. 

How does this sound to you?